ext/oj/dump.c

```diff
@@ -535,17 +535,17 @@ void oj_dump_xml_time(VALUE obj, Out out) {
     }
     if ((0 == nsec && !out->opts->sec_prec_set) || 0 == out->opts->sec_prec) {
         if (0 == tzsecs && rb_funcall2(obj, oj_utcq_id, 0, 0)) {
-            sprintf(buf,
+            int len = sprintf(buf,
                     "%04d-%02d-%02dT%02d:%02d:%02dZ",
                     ti.year,
                     ti.mon,
                     ti.day,
                     ti.hour,
                     ti.min,
                     ti.sec);
-            oj_dump_cstr(buf, 20, 0, 0, out);
+            oj_dump_cstr(buf, len, 0, 0, out);
         } else {
-            sprintf(buf,
+            int len = sprintf(buf,
```

Comment:
Same here, no need for a variable when the value will always be the same.

Reply:
Same above reply